command/server/listener.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/hashicorp/vault/helper/proxyutil"
 	"github.com/hashicorp/vault/helper/reload"
 	"github.com/hashicorp/vault/helper/tlsutil"
+	badcipher "github.com/jefferai/isbadcipher"
```

Comment:
Don't forget to add this though. Also why change the package name?

Reply:
Thanks. Fixed